Java/LocalDateTime.md

```diff
@@ -12,9 +12,7 @@ Camel’s case in java programming consists of compound words or phrases such th
 ## Methods in LocalDateTime and Naming Conventions
 | Method Name | Description and Naming Convention |
 |---|---|
```

Comment:
If you want the file to be a markdown file as you put .md extension, 
There is a better way to write it, go to the **Writing on Github section**, which you will find on the left navbar
https://docs.github.com/en/get-started
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/about-writing-and-formatting-on-github

Change this in other places too if you have.

Reply:
Done !